src/load_inputs/load_generators_data.jl

```diff
@@ -95,20 +97,35 @@ function load_generators_data!(setup::Dict, path::AbstractString, inputs_gen::Di
 		inputs_gen["RSV"] = gen_in[(gen_in[!,:Rsv_Max].>0),:R_ID]
 	end
 
+    NEW_BUILD = gen_in[gen_in.New_Build.==1, :R_ID]
```

Comment:
there seems to be some inconsistency within the code in general with tabs vs spaces. what is the standard? and can we get an auto formatter to address this as part of pre commit hooks?

Reply:
Prefer four spaces in new code. 
Yes, but let RSE do the reformat and figure out how to `git blame` past this.